app/assets/stylesheets/config/_import.scss

```diff
@@ -48,6 +48,8 @@ $colors: (
 
 $header-height: 56px;
 $site-width: 1280px;
+$cover-image-width: 1000px;
```

Comment:
question for my curious mind: why do we make this width static instead of variable (i.e. a percentage of the viewport or something similar). How do we come up with this value, is it linked to another value?

Reply:
It's the size of the cover image that someone came up with, I don't know what was the reasoning for these number, maybe @benhalpern will know.

Even though we do define this width & height here, I make sure that wherever we display cover image it is actually responsive so it doesn't matter what size of the container is.

Also, setting exact px size will help users create artworks: so you can simply tell them "_create a cover photo with dimensions at least 1000px x 420px_" instead of "_create a cover photo with size ratio 50:21 width to height_" :D